<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>fiets24.png</normaloff>fiets24.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="labelCurrentPlaylist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Playlist Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCurrentSong">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Song Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelcurrentAlbum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Album</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelcurrentArtist">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Artist Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTimeSong">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Time Song</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTotalTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Total Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNumberOfTracksAdded">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Number of tracks added</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="4" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="labelArtWork">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>174</width>
          <height>174</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Artwork Album</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxRepeatAll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Repeat All</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/48/media-playlist-repeat.svg</normaloff>:/icons/48/media-playlist-repeat.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxPlayShuffle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Play Shuffle</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/48/media-playlist-shuffle.svg</normaloff>:/icons/48/media-playlist-shuffle.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBarSong">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="format">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSliderSong">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="5">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidgetCurrentPlaylist">
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Artist</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Album</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Year</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Track</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Genre</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Duration</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Bitrate</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Samplerate</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Channels</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File location</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pushButtonPrevious">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/48/media-skip-backward.svg</normaloff>:/icons/48/media-skip-backward.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPlay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/48/media-playback-start.svg</normaloff>:/icons/48/media-playback-start.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNext">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/48/media-skip-forward.svg</normaloff>:/icons/48/media-skip-forward.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPause">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/48/media-playback-pause.svg</normaloff>:/icons/48/media-playback-pause.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonStop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/48/media-playback-stop.svg</normaloff>:/icons/48/media-playback-stop.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="labelcupll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Current Playlist:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelcuso">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Current Song:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelcual">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Current Album:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelcuar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Current Artist:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labeltiso">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Time Song:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labeltoti">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Total Time Playlist:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Total Number of Songs:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="labelVolume">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSliderVolume">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManagement">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="actionManagement"/>
    <addaction name="actionSave_Playlist"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add </string>
    </property>
    <addaction name="actionAddFile"/>
    <addaction name="actionAddFolder"/>
    <addaction name="actionOther_Playlist"/>
   </widget>
   <widget class="QMenu" name="menuRemove">
    <property name="title">
     <string>Remove</string>
    </property>
    <addaction name="actionDeleteTrack"/>
    <addaction name="actionDeletePlaylist"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_Window"/>
   </widget>
   <widget class="QMenu" name="menuSort">
    <property name="title">
     <string>Sort</string>
    </property>
    <addaction name="actionby_Artist"/>
    <addaction name="actionby_Album"/>
    <addaction name="actionby_Album_Year"/>
    <addaction name="actionby_Genre"/>
    <addaction name="actionby_Tracknumber"/>
    <addaction name="actionRandom"/>
    <addaction name="actionundo_Sort"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionOled_Display"/>
   </widget>
   <widget class="QMenu" name="menuSearchfilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionSearchfilter"/>
    <addaction name="actionby_Database"/>
   </widget>
   <addaction name="menuManagement"/>
   <addaction name="menuAdd"/>
   <addaction name="menuRemove"/>
   <addaction name="menuSort"/>
   <addaction name="menuSearchfilter"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/cancel.png</normaloff>:/icons/actions/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAddFile">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/document-open.png</normaloff>:/icons/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Music File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/folder-new.png</normaloff>:/icons/actions/folder-new.png</iconset>
   </property>
   <property name="text">
    <string>Music Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionOther_Playlist">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/legacy/list-add.png</normaloff>:/icons/legacy/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Other Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDeleteTrack">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/edit-delete.png</normaloff>:/icons/actions/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>selected Song(s)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionDeletePlaylist">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/legacy/list-remove.png</normaloff>:/icons/legacy/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>this Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionby_Artist">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/mail-forward.png</normaloff>:/icons/actions/mail-forward.png</iconset>
   </property>
   <property name="text">
    <string>by Artist</string>
   </property>
  </action>
  <action name="actionby_Album_Year">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/mail-forward.png</normaloff>:/icons/actions/mail-forward.png</iconset>
   </property>
   <property name="text">
    <string>by Album Year</string>
   </property>
  </action>
  <action name="actionby_Genre">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/mail-forward.png</normaloff>:/icons/actions/mail-forward.png</iconset>
   </property>
   <property name="text">
    <string>by Genre</string>
   </property>
  </action>
  <action name="actionOled_Display">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/configure.png</normaloff>:/icons/actions/configure.png</iconset>
   </property>
   <property name="text">
    <string>Oled and RTC</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionundo_Sort">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/edit-undo.png</normaloff>:/icons/actions/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>undo Sort</string>
   </property>
  </action>
  <action name="actionHelp_Window">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/help-contents.png</normaloff>:/icons/actions/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>Help Window</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionManagement">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/document-new.png</normaloff>:/icons/actions/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Management</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionby_Album">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/mail-forward.png</normaloff>:/icons/actions/mail-forward.png</iconset>
   </property>
   <property name="text">
    <string>by Album</string>
   </property>
  </action>
  <action name="actionSearchfilter">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/edit-find.png</normaloff>:/icons/actions/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>Filter Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSave_Playlist">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/document-save.png</normaloff>:/icons/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save Playlist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionby_Database">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/edit-undo.png</normaloff>:/icons/actions/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>undo Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/mail-reply-all.png</normaloff>:/icons/actions/mail-reply-all.png</iconset>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </action>
  <action name="actionby_Tracknumber">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/actions/mail-forward.png</normaloff>:/icons/actions/mail-forward.png</iconset>
   </property>
   <property name="text">
    <string>by Tracknumber</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButtonPlay</tabstop>
  <tabstop>pushButtonStop</tabstop>
  <tabstop>pushButtonPrevious</tabstop>
  <tabstop>pushButtonNext</tabstop>
  <tabstop>pushButtonPause</tabstop>
  <tabstop>horizontalSliderVolume</tabstop>
  <tabstop>checkBoxRepeatAll</tabstop>
  <tabstop>checkBoxPlayShuffle</tabstop>
  <tabstop>horizontalSliderSong</tabstop>
  <tabstop>tableWidgetCurrentPlaylist</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
